data/cmd-palete.ts

```diff
@@ -0,0 +1,125 @@
+import { IconType } from 'react-icons/lib'
```

Comment:
It's not clear why this (or other files exist in the `data` folder. I can't tell if this is related to Keystone cms, next or something homegrown.

Reply:
Okay so this more of a place holder folder until I finish setting up Keystone js and migrate most of this content over and not sure if I leave the cmd-palete.ts there